magda-scala-common/src/main/resources/common.conf

```diff
@@ -7,7 +7,7 @@ elasticSearch {
     }
 
     datasets {
-      version = 32
+      version = 34
```

Comment:
33?

Reply:
Bump to 34 just in case you merged your branch to master earlier
Will revert back to 33